Dockerfile

```diff
@@ -3,69 +3,77 @@
 
 FROM node:8.9.1-alpine as ghost-builder
 
-RUN \
-    apk update && apk upgrade                           && \
-    echo                                                && \
-    echo "--- Install ghost-cli --- "; echo             && \
-    npm install --loglevel=error -g ghost-cli           ;
-
-ENV GHOST_VERSION="1.17.1"                              \
-    GHOST_INSTALL="/var/lib/ghost"                      \
-    GHOST_CONTENT="/var/lib/ghost/content"              \
+ENV GHOST_VERSION="1.17.2"                                  \
+    GHOST_INSTALL="/var/lib/ghost"                          \
+    GHOST_CONTENT="/var/lib/ghost/content"                  \
     GHOST_USER="node"
 
 # Set default directory
 WORKDIR $GHOST_INSTALL
 
-# Run SQLite as database
-RUN \
-    ghost install "$GHOST_VERSION" \
-        --db sqlite3 --no-prompt \
-        --no-stack --no-setup \
-        --dir "$GHOST_INSTALL"                          && \
-    ghost config --ip 0.0.0.0 \
-        --port 2368 --no-prompt --db sqlite3 \
-        --url http://localhost:2368 \
-        --dbpath "$GHOST_CONTENT/data/ghost.db"         && \
-    ghost config paths.contentPath "$GHOST_CONTENT"     ;
-
+# We use SQLite as our DB
+RUN set -ex                                                 && \
+    apk update && apk upgrade                               && \
+    apk add --no-cache tzdata                               && \
+    cp /usr/share/zoneinfo/America/New_York /etc/localtime  && \
```

Comment:
I think we need to find a better way to set the timezone. Maybe with an environment variable?

Reply:
Of course, via a VAR it would make sense :)
But the issue is still there. The logs from the container don't consider the local time of the container.